docs/content/integrations/dbt/reference.mdx

```diff
@@ -131,62 +133,179 @@ Refer to the [Schedule documentation](/concepts/partitions-schedules-sensors/sch
 
 ---
 
-## Understanding asset keys
+## Understanding asset definition attributes generated for dbt nodes
```

Comment:
Are we ok with describing these Dagster properties as "Asset definition attributes"? I didn't want to use metadata since that term is overloaded in Dagster and dbt space.

Reply:
Are there other attributes an asset can have that aren't `definition attributes`?

Reply:
This is a bit of a mouthful, maybe "Understanding asset definition attributes"? I do think we're somewhat obliged to call these things "attributes" due to the existence of the "with_attributes()" function.

The main alternative I see would be "properties", but I don't see a huge benefit there